cfme/test_framework/sprout/client.py

```diff
@@ -111,7 +111,8 @@ def provision_appliances(
             group=stream, provider=provider, lease_time=lease_time, ram=ram, cpu=cpu, count=count
         )
         wait_for(
-            lambda: self.call_method('request_check', str(request_id))['finished'], num_sec=300)
+            lambda: self.call_method('request_check', str(request_id))['finished'], num_sec=300,
```

Comment:
I agree, but I'd like to post that to a new PR and just get the message added here